packages/protocol/tasks/utils.ts

```diff
@@ -27,6 +29,72 @@ async function deployContract(
     return deployed;
 }
 
+function compileYulContract(contractPath: string): string {
```

Comment:
Do you think it will be helpful to provide a `pnpm compile:yul` command?

Reply:
This command might be useful later on, added.